### DISCUSSION TITLE
Batch edit/trim & merge save at same time possible??

### DISCUSSION THREAD

**Discussion by geogan** [mifi/lossless-cut#1662] [General]
Hi, maybe I'm confused but I couldn't get simple batch trim of three files and concatenation to work at all... when I batch joined them, it just joined the three original untrimmed files together and saved them, instead of joining the three edited files I had loaded in batch. ie.

1. Loaded 3 of same format & size files in batch.
2. Double click each one to load in editor. Did simple trim to beginning / end of each one (one segment each).
3. Use the "Merge/concatenate files..." option in batch section to join files into one.

The file saved was just three original full unedited files instead of the three edits?!

How do I join/save the three trimmed/edited batch files instead of originals?

![image](https://github.com/mifi/lossless-cut/assets/48016214/e8141823-bc6d-41b7-af15-eb110f09eab7)

**Comment by mifi** [comment]
Hi. Unfortunately you need to do this in multiple operations.
1. First select the segments you want from each file (like you did)
2. export each file in the batch list (separately)
3. open the three exported files in the batch list (replace those you already have there)
4. now click the merge icon

I hope to improve this flow in the future.

**Reply by geogan** [reply]
Ok, that's what I did do eventually yes - reloaded the edits as batch and joined. Thought there might have been an easier built in way to do both operations at once. Would make sense that the edited versions would then be used for batch join instead of the original files.
Thanks for reply.
